test/inductor/transformer_model.py

```diff
@@ -0,0 +1,418 @@
+"""
+Below code is copied from gpt-fast:
+https://github.com/pytorch-labs/gpt-fast/blob/main/model.py
+"""
+
+import math
+from dataclasses import dataclass
+from typing import Optional
+
+import torch
+import torch.nn as nn
+from torch import Tensor
+
```

Comment:
I really don't like these tests that commit the full model... it's just harder to debug when something goes wrong. this should be a test in gpt fast not here